python/ray/rllib/ddpg/common/schedules.py

```diff
@@ -0,0 +1,108 @@
+"""This file is used for specifying various schedules that evolve over
```

Comment:
This file is literally a copy of `dqn/schedules.py`. Let's move that to common to avoid this code duplication.

Reply:
To avoid making any change to dqn, I just use the schedulers of dqn currently.